content/docs/rfcs/70/README.md

```diff
@@ -0,0 +1,236 @@
+---
+slug: 70
+title: 70/ETH-SECPM
+name: Private 1:1 messages over Ethereum
```

Comment:
The aim of the RFC is not to "Send message over Ethereum" but to setup a secure channel based on the cryptographic data available to Ethereum users right?

I'd recommend to change the title to:

> Secure channel setup using Ethereum account

Or

> Private 1:1 messages for Ethereum users

or anything in this domain.